finite_state_sdk/queries.py

```diff
@@ -377,26 +418,50 @@ def _create_GET_FINDINGS_VARIABLES(asset_version_id=None, category=None, cve_id=
             }
         )
 
+    if status is not None:
+        variables["filter"]["currentStatus"] = {
+            "status_in": [
+                status
+            ]
+        }
+
     return variables
 
 
+GET_FINDINGS_COUNT = {
+    "query": """
+query GetFindingsCount($filter: FindingFilter)
+{
+    _allFindingsMeta(filter: $filter) {
+        count
+    }
+}
+""",
+    "variables": lambda asset_version_id=None, category=None, cve_id=None, status=None, severity=None, limit=None: _create_GET_FINDINGS_VARIABLES(asset_version_id=asset_version_id, category=category, cve_id=cve_id, status=status, severity=severity, limit=limit, count=True)
```

Comment:
Consider separating the variable assignments from the lambda expression for better readability and maintainability.

```python
def get_findings_variables(asset_version_id=None, category=None, cve_id=None, status=None, severity=None, limit=None):
    return _create_GET_FINDINGS_VARIABLES(
        asset_version_id=asset_version_id,
        category=category,
        cve_id=cve_id,
        status=status,
        severity=severity,
        limit=limit,
        count=True
    )
```

Reply:
Also a good suggestion - we can get this in a future release to improve readability for all these functions